aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunTestUtility.java

```diff
@@ -56,4 +61,28 @@ public static AliyunOSSTestRule initialize() {
 
     return testRule;
   }
+
+  public static String testOSSEndpoint() {
```

Comment:
Nit:  do we still need the `test` prefix for the method names ? I see it is already in an AliyunTestUtility.